Yoshi/Yoshi/View Controllers/Debug View Controller/DebugViewController.swift

```diff
@@ -128,8 +128,25 @@ extension DebugViewController: UITableViewDataSource {
 
         let option = options[indexPath.row]
         cell.textLabel?.text = option.title
-        cell.detailTextLabel?.text = option.subtitle
-
+        
+        if let subtitle = option.subtitle {
+            cell.detailTextLabel?.text = subtitle
+        } else {
+            switch option {
+            case let dateSelectorMenu as YoshiDateSelectorMenu:
+                cell.detailTextLabel?.text = dateSelectorMenu.selectedDate.description
```

Comment:
We may want to introduce a date formatter, the current description style (e.g., 2016-04-27 21:34:02 +0000), is not very readable